modeling-cmds/src/ok_response.rs

```diff
@@ -766,44 +766,57 @@ define_ok_modeling_cmd_response_enum! {
             pub max_distance: LengthUnit,
         }
 
+        /// Faces and edges id info (most used in identifying geometry in patterned and mirrored objects).
+        #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, ModelingCmdOutput)]
+        pub struct FaceEdgeInfo {
+            /// The UUID of the object.
+            pub object_id: Uuid,
+            /// The faces of each object.
+            pub faces: Vec<Uuid>,
+            /// The edges of each object.
+            pub edges: Vec<Uuid>,
+        }
+
         /// The response from the `EntityClone` command.
         #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, ModelingCmdOutput)]
         pub struct EntityClone {
+            /// The Face and Edge Ids of the cloned entity.
+            pub face_edge_ids: Vec<FaceEdgeInfo>,
         }
 
         /// The response from the `EntityLinearPatternTransform` command.
         #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, ModelingCmdOutput)]
         pub struct EntityLinearPatternTransform {
-            /// The UUIDs of the entities that were created.
-            pub entity_ids: Vec<Uuid>,
+            /// The Face, edge, and entity ids of the patterned entities.
+            pub entity_face_edge_ids: Vec<FaceEdgeInfo>,
```

Comment:
@adamchalmers do you know how we can make these bw compatible so we dont break in flux releases or should we just slow roll it out w empty vecs?